src/EnergyPlus/ThermalComfort.cc

```diff
@@ -2853,13 +2856,10 @@ namespace ThermalComfort {
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         std::string epwLine;
-        static Real64 avgDryBulbCEN(0.0);
         Real64 dryBulb;
         Real64 tComf;
         Real64 tComfLow;
-        static Real64 runningAverageCEN(0.0);
         Real64 numOccupants;
-        static bool useEpwData(false);
```

Comment:
being made global in the previous pull request concerning ASH55

Reply:
I just want to make sure I'm clear of the intent here.  So this should've been removed in the previous PR?

Reply:
There used to be one of this variable in both ASH55 and CEN15251 separately, and the one in ASH55 was made global in the precious PR for the interest of unit tests (as I understand). So I think this one is redundant in the CEN15251, since there has been a shared global one.